src/main/java/io/workflow/bpmnsimulator/fieldvalidator/AssigneeFieldValidator.java

```diff
@@ -0,0 +1,29 @@
+package io.workflow.bpmnsimulator.fieldvalidator;
+
+import io.workflow.bpmnsimulator.model.Field;
+import io.workflow.bpmnsimulator.model.Step;
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.engine.task.Task;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+@Slf4j
+@Service
+class AssigneeFieldValidator implements FieldValidator {
+
+    @Override
+    public FieldValidationResult validate(@Nonnull final Step step, @Nonnull final Task task) {
+        final boolean isValid = Objects.equals(task.getAssignee(), step.getAssignee());
```

Comment:
task can be multi instance and assignee is collection of item

Reply:
I've created a ticket for changing assignee to a collection of String: https://github.com/mohsen-ebrahimi/bpmn-simulator/issues/7
But I didn't get what you mean by `task can be multi-instance`. Let's discuss it here: